testplan/runnable/interactive/base.py

```diff
@@ -662,7 +662,7 @@ def _initial_report(self):
             for suite in test.suites:
                 suite_name = suite.__class__.__name__
                 suite_report = testplan.report.TestGroupReport(
-                    name=suite_name, uid=suite_name, category="suite"
+                    name=suite_name, uid=suite_name, category='suite'
```

Comment:
Why have you made this change? Double quotes are the standard used by `black`.

Reply:
just for consistency with context